packages/@okta/vuepress-site/docs/reference/api/devices/index.md

```diff
@@ -1182,9 +1198,33 @@ The following diagram shows the state object for a Device:
         "meid": null,
         "udid": "36A56558-1793-5B3A-8362-ECBAA14EDD2D",
         "sid": null,
-        "tpmPublicKeyHash":null,
-        "registered":true,
-        "secureHardwarePresent":false
+        "tpmPublicKeyHash": null,
+        "registered": true,
+        "secureHardwarePresent": false,
+        "diskEncryptionType": null
```

Comment:
Is `null` an acceptable value for `diskEncryptionType`? It's not defined in the table

Reply:
I assumed it was, however, that is a good question. @apoorvadeshpande-okta could you confirm?

Reply:
Usually `null` values are not shown on the API response over http, but for the sake of an example please populate the correct value for macOS platform in this case